docs/ballot.md

```diff
@@ -0,0 +1,26 @@
+---
+title: Ballot Operation
+id: ballot
+author: Davis Sawali
+---
+
+
+The `Ballot` operation allows delegates to cast one `Yay`, `Nay`, or `Pass` ballot on a selected proposal. Delegates are only able to cast their votes during the **Exploration period**.
```

Comment:
Delegates can vote during the Promotion period too. https://tezos.gitlab.io/active/voting.html